tests/callbacks/test_data_monitor.py

```diff
@@ -38,45 +55,71 @@ def test_base_log_interval_override(log_histogram, tmpdir, log_every_n_steps, ma
     ],
 )
 @mock.patch("pl_bolts.callbacks.data_monitor.DataMonitorBase.log_histogram")
-def test_base_log_interval_fallback(log_histogram, tmpdir, log_every_n_steps, max_steps, expected_calls, datadir):
+def test_base_log_interval_fallback(
+    log_histogram,
+    tmpdir,
+    log_every_n_steps,
+    max_steps,
+    expected_calls,
+    datadir,
+    catch_warnings,
+):
     """Test that if log_every_n_steps not set in the callback, fallback to what is defined in the Trainer."""
+    warnings.filterwarnings(
+        "ignore",
+        message=".*does not have many workers which may be a bottleneck.*",
+        category=PossibleUserWarning,
+    )
     monitor = TrainingDataMonitor()
     model = LitMNIST(data_dir=datadir, num_workers=0)
     trainer = Trainer(
         default_root_dir=tmpdir,
         log_every_n_steps=log_every_n_steps,
         max_steps=max_steps,
         callbacks=[monitor],
+        accelerator="auto",
     )
     trainer.fit(model)
     assert log_histogram.call_count == (expected_calls * 2)  # 2 tensors per log call
 
 
-def test_base_no_logger_warning():
+def test_base_no_logger_warning(catch_warnings):
     """Test a warning is displayed when Trainer has no logger."""
     monitor = TrainingDataMonitor()
-    trainer = Trainer(logger=False, callbacks=[monitor])
+    trainer = Trainer(logger=False, callbacks=[monitor], accelerator="auto", max_epochs=-1)
     with pytest.warns(UserWarning, match="Cannot log histograms because Trainer has no logger"):
-        monitor.on_train_start(trainer, pl_module=None)
+        monitor.on_train_start(trainer, pl_module=LightningModule())
 
 
-def test_base_unsupported_logger_warning(tmpdir):
+def test_base_unsupported_logger_warning(tmpdir, catch_warnings):
     """Test a warning is displayed when an unsupported logger is used."""
+    warnings.filterwarnings(
+        "ignore",
+        message=".*is deprecated in v1.6.*",
+        category=LightningDeprecationWarning,
+    )
```

Comment:
I think you should avoid the deprecated usage here and pass in a list instead.

Reply:
@luca-medeiros, I missed this one, can you please resolve it? @awaelchli is absolutely correct that we don't want to be silencing `LightningDeprecationWarning`s

Reply:
Yea, actually while adding this I actually wasn't sure about the best approach to it.
Thanks for reminding!